hazelcast/src/test/java/com/hazelcast/cluster/ClusterShutdownTest.java

```diff
@@ -56,11 +57,19 @@ private void testClusterShutdownWithSingleMember(final ClusterState clusterState
         final Thread thread = new Thread(new Runnable() {
             @Override
             public void run() {
-                instances[0].getCluster().shutdown();
+                while (true) {
+                    try {
+                        instances[0].getCluster().shutdown();
+                        break;
+                    } catch (Exception ignored) {
```

Comment:
It is a cluster shutdown which terminates all nodes one by one. It will not fail while actually shutting down the nodes, it will fail during moving the cluster to the shut-down-able state. Since we know it will fail because of replica sync operations, I ignored the exception. How can a cluster shutdown operation fail is well-documented in javadocs.